docs/integrations/spark.md

```diff
@@ -24,17 +24,22 @@ Similarly, properties defining lakeFS credentials should be configured in secure
 not on the command line or inlined in code where they might be exposed.
 {: .note}
 
-## Two-tiered Spark support
+## Multi-tiered Spark support
 
-lakeFS support in Spark has two tiers:
+lakeFS support in Spark has three tiers:
 
 * Access lakeFS using the [S3A gateway](#access-lakefs-using-the-s3a-gateway).
 * Access lakeFS using the [lakeFS-specific Hadoop
   FileSystem](#access-lakefs-using-the-lakefs-specific-hadoop-filesystem).
+* (**BETA!**) Access lakeFS using lake lakeFS-specific Hadoop FileSystem and
```

Comment:
I like the fact that you are adding lakeFSFS + FOC as another support tier and not as a configuration option of lakeFSFS!

Reply:
** *For now!* **  

![One day all this will be the default](https://user-images.githubusercontent.com/7802932/200159298-dac85e7a-5536-45cb-8bfc-6dc2388ae714.png)